src/main/java/org/jitsi/jicofo/discovery/DiscoveryUtil.java

```diff
@@ -81,6 +81,11 @@
      */
     public final static String FEATURE_DTLS = "urn:xmpp:jingle:apps:dtls:0";
 
+    /**
+     * Source name signalling required for the multi-stream mode.
+     */
+    public final static String FEATURE_SOURCE_NAMES = "'http://jitsi.org/source-name'";
```

Comment:
Are the quotes intentional (both `"` and `'`)?

Reply:
no, but since this was not tested yet, that's why it wouldn't work I guess :) thanks